packages/battery_plus/battery_plus/example/integration_test/battery_plus_test.dart

```diff
@@ -18,11 +18,11 @@ void main() {
   testWidgets('Can get battery level', (WidgetTester tester) async {
     final batteryLevel = await Battery().batteryLevel;
     expect(batteryLevel, isNotNull);
-  }, skip: !Platform.isAndroid);
+  }, skip: !Platform.isAndroid && !Platform.isMacOS);
 
   testWidgets('Can get if device is in battery save mode',
       (WidgetTester tester) async {
     final isInBatterySaveMode = await Battery().isInBatterySaveMode;
     expect(isInBatterySaveMode, false);
-  });
+  }, skip: !Platform.isAndroid && !Platform.isIOS && !Platform.isWindows);
```

Comment:
Could you document why this tests needs to be skipped for MacOS?

Maybe it is better to write `skip: Platform.isMacOs` instead.

Reply:
i'll document this in the getter itself. fyi: this feature is not implemented in MacOS yet.